packages/client/components/ErrorBoundary.tsx

```diff
@@ -25,21 +23,11 @@ class ErrorBoundary extends Component<Props, State> {
   }
 
   componentDidCatch(error: Error, errorInfo: ErrorInfo) {
-    const {atmosphere} = this.props
-    const {viewerId} = atmosphere
-    const store = atmosphere.getStore()
-    const email = (store as any)?._recordSource?._records?.[viewerId]?.email ?? ''
```

Comment:
I don't think this was working, in general, before:

![Screen Shot 2022-02-01 at 2 03 27 PM](https://user-images.githubusercontent.com/630449/152074410-dad11b57-e452-4a5d-8d52-5fb43a5807c0.png)

because `_records` is a `Map`, you have to use `.get` as opposed to `[]` syntax.